plugins/core/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qAdditionalIO )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qAnimation )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qBroom )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qCompass )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qCork )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qCSF )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qCSVMatrixIO )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qEDL )
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qExampleGLPlugin )
```

Comment:
Why not keeping the automated way as before? And do we really want to add the example plugins?

Reply:
CMake recommends against using GLOBs/auto-inclusion because some build systems don't get alerted about changes. Since this directory won't be changing that frequently I went to explicit adding of directories. The "auto-add" idea will be used in the 3rdParty directory to make it easy for developers to add their plugins there.

About the examples: That's part of step 2. They'll live in the 3rdParty directory (even though they aren't' really 3rd party).

Reply:
...or maybe an examples directory? That might make more sense.

Reply:
Is this a deal-breaker? :-)

From the CMake [manual](https://cmake.org/cmake/help/v3.3/command/file.html):
> We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.

Reply:
OK thanks. The only two things that need changing per plugin should be:

-include( ../CMakePluginTpl.cmake )
+include( ../../CMakePluginTpl.cmake )

in the CMakeLists.txt file and removing relative paths:

-#include "../ccStdPluginInterface.h"
+#include "ccStdPluginInterface.h"

I changed all the plugins in the repo, so it should just be your "private" ones. CANUPO!